templates/services/merge/validator/lighthouse.tmpl

```diff
@@ -1,36 +1,10 @@
 {{/* lighthouse.tmpl */}}
 {{ define "validator" }}
-  validator-import:
-    container_name: validator-import-client
-    build:
-      context: github.com/NethermindEth/lighthouse-init-validator
-      args:
-        LH_VERSION: ${VL_IMAGE_VERSION}
-        NETWORK: {{if .VlCustomCfg}}CUSTOM{{else}}{{if .SplittedNetwork}}${CL_NETWORK}{{else}}${NETWORK}{{end}}{{end}}{{with .VlCustomCfg}}
-    depends_on:
-      config_consensus:
-        condition: service_completed_successfully{{end}}
-    networks:
-      - sedge
-    volumes:
-      - ${KEYSTORE_DIR}:/keystore
-      - ${VL_DATA_DIR}:/data{{if .VlCustomCfg}}{{if .VlRemoteCfg}}
-      - ./config.yml:/network_config/config.yaml{{end}}{{if .VlRemoteGen}}
-      - ./genesis.ssz:/network_config/genesis.ssz{{end}}{{if .VlRemoteDpl}}
-      - ./deploy_block.txt:/network_config/deploy_block.txt{{end}}{{end}}{{if .LoggingDriver}}
```

Comment:
I don't see these volumes being set up in `setupLighthouseValidatorImport`

Reply:
Please check this commit https://github.com/NethermindEth/sedge/pull/208/commits/09743fd0d382943870ce2066823bbcdded15964a

Reply:
I don't see the validator data dir being mounted

Reply:
Sorry, I should have been more specific. Notice [these lines](https://github.com/NethermindEth/sedge/blob/a981c5d0edc8fab4c00132be5f3a111295716181/cli/actions/importKeys.go#L288-L291). On those lines, all validator client volumes are also mounted by the import-keys container using the `VolumesFrom` property, that implies the validator data dir is mounted.